pages/404.tsx

```diff
@@ -1,64 +1,80 @@
 /*
 * @2023 Open Seattle
 */
-import { Box, Container, Link, Typography, useTheme } from '@mui/material'
+import { Box, Button, Container, Typography, useMediaQuery, useTheme } from '@mui/material'
 import SectionContainer from 'components/layout/SectionContainer'
 import { withBasicLayout } from 'components/layouts'
 import ErrorImage from '../assets/404-error.png'
 
-function Page404() {
+// Promote, if we want to handle other error types
```

Comment:
What do you mean here?

Reply:
Agree there could be other error, like the airtable for volunteer and partner forms don't work.

Reply:
Those won't be 404 errors. We'd need to replace those forms with something else. It has happened before that airtable was down for a bit.